org.gnuhealth.mygnuhealth.yaml

```diff
@@ -0,0 +1,31 @@
+app-id: org.gnuhealth.mygnuhealth
+base: io.qt.qtwebkit.BaseApp
+base-version: '5.15-21.08'
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.llvm12
+command: mygnuhealth
+rename-icon: mygnuhealth
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=network
+  - --share=ipc
+cleanup:
+  - '/share/doc'
+modules:
+  - python3-pyside2.yaml
+  - python3-matplotlib.yaml
+  - python3-bcrypt.yaml
+  - python3-tinydb.yaml
+  - python3-requests.yaml
+  - name: mygnuhealth
+    buildsystem: simple
+    build-commands:
+      - pip3 install --prefix=/app --no-deps .
+    sources:
+      - type: archive
+        url: http://hg.savannah.gnu.org/hgweb/health-mygnuhealth/archive/tip.tar.gz
```

Comment:
This is an ever changing content. You should pin a specific revision / tag.

Reply:
Yes. It was only a test.
Probably upstream will release version 1.0.6 soon, following the proposed changes to appdata and desktop file.